src/go/rpk/pkg/cli/cmd/cluster/config/status.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func newStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Get configuration status of redpanda nodes",
+		Long: `Get configuration status of redpanda nodes
+
+For each node, indicate which configuration version it is up to date with,
```

Comment:
What does `configuration version it is up to date with` mean? Is there a cluster-wide configuration sequence number internally, and each edit bumps this number? If so, can this be explained in (hopefully just one) a sentence?

Reply:
Tweaked this text a bit to clarify